### ISSUE TITLE
[waytowich2018] Compact Convolutional Neural Networks for Classification of Asynchronous Steady-state Visual Evoked Potentials

### ISSUE THREAD

**Issue by mxochicale** [mxochicale/arXiv#23]
## ? Google Scholar Citations by date?
arXiv: https://arxiv.org/abs/1803.04566
PDF: https://arxiv.org/pdf/1803.04566.pdf

# BibTex-File
```
@article{article,
  author  = {}, 
  title   = {},
  journal = {},
  year    = {},
  number  = {},
  pages   = {},
  month   = {},
  volume  = {},
  issn = {},
  doi = {},
  url = {},
  keywords = {},
  abstract = {}
}
```

# Compact Convolutional Neural Networks for Classification of Asynchronous Steady-state Visual Evoked Potentials

## Author(s) Surname Foreanem / Website

Waytowhich Nicolas R
https://arxiv.org/find/cs/1/au:+Waytowich_N/0/1/0/all/0/1
 
Lawhern Vernon 
https://arxiv.org/find/cs/1/au:+Lawhern_V/0/1/0/all/0/1

Garcia Javier O. 
https://arxiv.org/find/cs/1/au:+Garcia_J/0/1/0/all/0/1

Cummings Jennifer 
https://arxiv.org/find/cs/1/au:+Cummings_J/0/1/0/all/0/1

Faller Josef 
https://arxiv.org/find/cs/1/au:+Faller_J/0/1/0/all/0/1

Sajda Paul 
https://arxiv.org/find/cs/1/au:+Sajda_P/0/1/0/all/0/1

Vettel Jean M. 
https://arxiv.org/find/cs/1/au:+Vettel_J/0/1/0/all/0/1

## Abstract
Steady-State Visual Evoked Potentials (SSVEPs) are neural oscillations from the parietal and occipital regions of the brain that are evoked from flickering visual stimuli. SSVEPs are robust signals measurable in the electroencephalogram (EEG) and are commonly used in brain-computer interfaces (BCIs). However, methods for high-accuracy decoding of SSVEPs usually require hand-crafted approaches that leverage domain-specific knowledge of the stimulus signals, such as specific temporal frequencies in the visual stimuli and their relative spatial arrangement. When this knowledge is unavailable, such as when SSVEP signals are acquired asynchronously, such approaches tend to fail. In this paper, we show how a compact convolutional neural network (Compact-CNN), which only requires raw EEG signals for automatic feature extraction, can be used to decode signals from a 12-class SSVEP dataset without the need for any domain-specific knowledge or calibration data. We report across subject mean accuracy of approximately 80% (chance being 8.3%) and show this is substantially better than current state-of-the-art hand-crafted approaches using canonical correlation analysis (CCA) and Combined-CCA. Furthermore, we analyze our Compact-CNN to examine the underlying feature representation, discovering that the deep learner extracts additional phase and amplitude related features associated with the structure of the dataset. We discuss how our Compact-CNN shows promise for BCI applications that allow users to freely gaze/attend to any stimulus at any time (e.g., asynchronous BCI) as well as provides a method for analyzing SSVEP signals in a way that might augment our understanding about the basic processing in the visual cortex.

## Results

# RAW COMMENTS on 15 march 2018

EEG signals can be analysed using Power Spectral Density Analysis and using  
Multivariable statistical analysis technique called Canonical correlation analysis (CCA)  
\cite{lin2007, bin2009, waytowich2017}
and the most recently work of combined-CCA \cite{nakanishi2014, chen2015, waytowich2016}.
Even with those improvements, the latter requires "user-specific calibration data to construct reference signals." 

waytowich2018 et al. prosed the use of a four layer compact-CNN to learnt invariant features of raw EEG signals and train the networks with a relative small datasets.

## Model architecture for the compact-CNN
"each convolution layer is followed by batch normalization, 2Daverage pooling, and 2D spatial dropout layers.  
* In the first three layers, we use the exponential linear unit (ELU) non-linearity (as opposed to other non-linear activation functions such as sigmoidsor rectified linear units) as we found this to give superior performance for EEG classification.
*The  fourth  and  final  layer  is  connected  to  a  dense  layer  with  a  softmax  activation  function  forclassification.  Our Compact-CNN is trained using a categorical cross-entropy loss function.

The model was implemented in Tensorflow, using the Keras API.  The model was trained for 500 iterations using the Adam optimizer, with a minibatch size of 64 trials.  The dropout probability was set to 0.25 for all layers.  For this application, we learn 96 temporal-spatial filter pairs (F= 96).  
"
